components/desktop/xchat/patches/xchat-05-perl-build.patch

```diff
@@ -0,0 +1,11 @@
+--- xchat-2.8.8/configure.in.1	2016-03-25 22:54:23.856575624 +0300
++++ xchat-2.8.8/configure.in	2016-03-25 22:55:05.410528251 +0300
+@@ -254,7 +254,7 @@
+ if test "$perl" = yes; then
+ 	AC_PATH_PROG(perlpath, perl)
+ 	AC_MSG_CHECKING(for Perl compile flags)
+-	PERL_CFLAGS=`$perlpath -MExtUtils::Embed -e ccopts 2>/dev/null`
++	PERL_CFLAGS=`$perlpath -MExtUtils::Embed -e ccopts 2>/dev/null | sed -e s:-fstack-protector-strong::`
```

Comment:
Thank you soo much for this!  This PERL_CFLAGS line fixed an issue I was having building a third party Pidgin plugin for Google Hangouts on OpenIndiana.  This saved me a lot of headache ;)